src/daemon/daemon_config.cpp

```diff
@@ -104,10 +104,22 @@ std::unique_ptr<const mp::DaemonConfig> mp::DaemonConfigBuilder::build()
     auto multiplexing_logger = std::make_shared<mpl::MultiplexingLogger>(std::move(logger));
     mpl::set_logger(multiplexing_logger);
 
+    auto storage_path = qgetenv("MULTIPASS_STORAGE");
```

Comment:
What if the directory isn't there, can't be created, doesn't have the necessary permissions? Or if the value is wrong entirely? Do we handle all those cases properly in the daemon?

Reply:
We throw:
```
multipassd[1527212]: Caught an unhandled exception: unable to create directory '/bad/path/cache/lxd'
```

Reply:
OK perfect.